include/wx/cmdargs.h

```diff
@@ -31,19 +31,17 @@ class WXDLLIMPEXP_BASE wxCmdLineArgsArray
     wxCmdLineArgsArray() { m_argsA = NULL; m_argsW = NULL; }
 
     template <typename T>
-    wxCmdLineArgsArray& operator=(T **argv)
```

Comment:
Is it all right to remove this operator with regard to user code? The, Unicode-only, `wxCmdLineArgsArray` class surely is very rarely used (let alone the operator), if ever, and it's not documented but the class is part of the public interface and the operator has been around for quite some years.